tests/ClientBuilderTest.php

```diff
@@ -67,7 +67,7 @@ public function testSetMessageFactory(): void
         /** @var MessageFactory|MockObject $messageFactory */
         $messageFactory = $this->createMock(MessageFactory::class);
 
-        $clientBuilder = new ClientBuilder(['dsn' => 'http://public:secret@example.com/sentry/1']);
+        $clientBuilder = new ClientBuilder(new Options(['dsn' => 'http://public:secret@example.com/sentry/1']));
```

Comment:
To simplify the tests I would use the `create` method so that we avoid creating the `Options` object everywhere